spec/models/dor_object_workflow_status_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'spec_helper'
+
+RSpec.describe DorObjectWorkflowStatus do
+  subject { described_class.new(pid) }
+  let(:pid) { 'druid:abc123' }
+  describe '#can_open_version?' do
+    context 'when not accessioned' do
+      before do
+        expect(Dor::WorkflowService)
+          .to receive(:get_lifecycle).with('dor', pid, 'accessioned')
+          .and_return(false)
+      end
+      it { expect(subject.can_open_version?).to eq false }
+    end
```

Comment:
Prefer all these `context` with one `it` are all converted to `it`s, like:
```
it 'is not open when not accessioned' do
  expect(Dor::WorkflowService).to receive(:get_lifecycle).with('dor', pid, 'accessioned').and_return(false)
  expect(subject.can_open_version?).to be false
end
```
(Long lines up to 120 chars should be OK.)